### PR TITLE
[MISC][SCHEMA] Add schema-code sphinx generated read the docs to CI; pushes to bids-standard/schema-docs

### PR DESCRIPTION
**Pull request by bendhouseart**
Setup sphinx to generate RTD page for schema-code and push changes to separate repo at `bids-standard/schema-docs` to trigger build on readthedocs page.

Need to follow up w/ creation of a RTD page as current page [schemadocsplaceholder](https://schemadocsplaceholder.readthedocs.io/en/latest/) is under my account  not bids-standard.

### CODE REVIEW

**Comment by effigies** [comment]
Is there a reason CI is failing?

**Comment by bendhouseart** [comment]
> Is there a reason CI is failing?

There is, but I don't know why. But if it's merged I can manually trigger it and debug.

File: .github/workflows/schema_docs.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+on:
+  push:
+    branches:
+      - "master"
+    tags:
+      - "schema-*"
+  pull_request:
+    branches:
+      - "*"
```
Comment (effigies) [inline .github/workflows/schema_docs.yaml:11]:
```suggestion
```

Comment (bendhouseart) [reply .github/workflows/schema_docs.yaml:11]: 👍

**Pull request merged**